### PR TITLE
Add i18n function and Japanese resource to streamlit chime

### PR DESCRIPTION
**Pull request by kochizufan**
### Link to issue
- No related issues. This is individual PR.

### Changelog entry
- Add i18n feature to streamlit-version of CHIME, with Japanese resource.
   To realize this, add python-i18n to dependencies.

### Additional info
- I know dash-version CHIME has i18n feature, so I will contribute to dash-version by adding Japanese resources later.
  But, I feel some performance issues for the dash-version, so I want to have Japanese version of streamlit CHIME.
  So I add i18n feature and Japanese resource to it.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,11 @@
 -e .
+streamlit
```
Comment (jlubken) [inline requirements.txt]: These do not go here. The `-e .` delegates to the dependencies listed in `setup.py`

**Review by jlubken — COMMENTED** [review]
I believe that something is missing from the python packaging to ensure that the locale yml files are installed as part of the python module.

File: requirements.txt (hunk 1 above)
Comment (kochizufan) [reply requirements.txt]: I added MANIFEST.in to solve the problem.
I confirmed it works in Docker.

Comment (kochizufan) [reply requirements.txt]: And I removed package names from requirements.txt

**Review by BrianThomasRoss — APPROVED** [review]
Thanks for this. Incredibly important work.

**Pull request merged**